docs/content/preview/explore/observability/active-session-history.md

```diff
@@ -28,27 +26,18 @@ Analyzing the wait events and wait event types lets you troubleshoot, answer the
 
 ## Configure ASH
 
-To use ASH, enable and configure the following flags for each node of your cluster.
+To configure ASH, you can set the following YB-TServer flags for each node of your cluster. Changing these flags doesn't require a VM restart.
 
-| Flag | Description |
-| :--- | :---------- |
-| allowed_preview_flags_csv | Set the value of this flag to include `ysql_yb_ash_enable_infra,ysql_yb_enable_ash`. |
-| ysql_yb_ash_enable_infra | Enable or disable ASH infrastructure. <br>Default: false. Changing this flag requires a VM restart. |
-| ysql_yb_enable_ash | Works only in conjunction with the flag `ysql_yb_ash_enable_infra`. Setting this flag to true enables the collection of wait events for YSQL and YCQL queries, and YB-TServer requests.<br> Default: false. Changing this flag doesn't require a VM restart. |
-
-### Additional flags
-
-You can also use the following flags based on your requirements.
-
-| Flag | Description |
-| :--- | :---------- |
-| ysql_yb_ash_circular_buffer_size | Size (in KBs) of circular buffer where the samples are stored. <br> Default: 16*1024. Changing this flag requires a VM restart. |
-| ysql_yb_ash_sampling_interval_ms | Sampling interval (in milliseconds). <br>Default: 1000. Changing this flag doesn't require a VM restart. |
-| ysql_yb_ash_sample_size | Maximum number of events captured per sampling interval. <br>Default: 500. Changing this flag doesn't require a VM restart. |
+| Flag | Description | Default |
+| :--- | :---------- | :------ |
+| ysql_yb_enable_ash | Set this flag to true enable the collection of wait events for YSQL and YCQL queries, and YB-TServer requests. | true |
```

Comment:
Statement is bit confusing. 
`Set this flag to true enable the collection of wait events for YSQL and YCQL queries, and YB-TServer requests. `